internal/testgen/gen_contains.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"strings"
+	"text/template"
+
+	"github.com/Antonboom/testifylint/internal/checkers"
+)
+
+type ContainsTestsGenerator struct{}
+
+func (ContainsTestsGenerator) Checker() checkers.Checker {
+	return checkers.NewContains()
+}
+
+func (g ContainsTestsGenerator) TemplateData() any {
+	checker := g.Checker().Name()
+
+	return struct {
+		CheckerName       CheckerName
+		InvalidAssertions []Assertion
+		ValidAssertions   []Assertion
+		IgnoredAssertions []Assertion
+	}{
+		CheckerName: CheckerName(checker),
+		InvalidAssertions: []Assertion{
+			{
+				Fn:            "True",
+				Argsf:         `strings.Contains(s, "abc123")`,
+				ReportMsgf:    checker + ": use %s.%s",
+				ProposedFn:    "Contains",
+				ProposedArgsf: "s, \"abc123\"",
```

Comment:
Usually this is written 

```suggestion
				ProposedArgsf: `s, "abc123"`,
```

For readability purpose